src/schemathesis/loaders.py

```diff
@@ -83,6 +85,12 @@ def from_pytest_fixture(
     return LazySchema(fixture_name, method=method, endpoint=endpoint, tag=tag)
 
 
+def from_wsgi(app, schema_path):
+    client = Client(app, BaseResponse)
+    response = client.get(schema_path)
+    return from_file(response.data)
```

Comment:
I propose to pass `client` to all other loaders and store it as an optional attribute on `BaseSchema`. We can pass it to `Case` instances later and then use it for calling the app via `case.wsgi_call`